src/VisualStudio/Directory.Build.targets

```diff
@@ -0,0 +1,22 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
Looks like you added this in your other PR, so you can remove it here if this gets folks confused looking at history.

Reply:
yeah.  this is forked off that branch, so ti won't be confusing.